docs/pages/database-access/guides/ha.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Database Access High Availability
+description: How to configure Teleport Database Access in HA configuration
+---
+
+# Database Access High Availability
+
+You can deploy Database Access in a highly available configuration in a couple
+of common ways: separate replicas and combined replicas. Both of those revolve
+around pointing multiple Database Services to the same database instance.
+
+## Separate Replicas
+
+With separate replicas, each Database Service agent proxying the database
+assigns it a different name. This allows you to explicitly pick the agent
+you want to connect to the database over.
+
+For example, you may have two Database Services deployed in different
+availability zones providing access to the same PostgreSQL database:
+
+```yaml
+# Database service agent #1.
+db_service:
+  enabled: "yes"
+  databases:
+  # Note the name is different than agent #2 but URI is the same.
+  - name: "postgres-us-east-1a"
+    protocol: "postgres"
+    uri: "postgres.example.com:5432"
+```
+
+```yaml
+# Database service agent #2.
+db_service:
+  enabled: "yes"
+  databases:
+  # Note the name is different than agent #1 but URI is the same.
+  - name: "postgres-us-east-1b"
+    protocol: "postgres"
+    uri: "postgres.example.com:5432"
+```
+
+With this configuration, both services will appear as two separate entries in
+`tsh db ls` output and you will have to pick one explicitly when connecting:
+
+```bash
+$ tsh db ls
+Name
+-------------------
+postgres-us-east-1a
+postgres-us-east-1b
+
+$ tsh db login postgres-us-east-1a
+$ tsh db connect postgres-us-east-1a
+```
+
+This approach is useful when you want to have control over which replica
+you're using to connect.
+
+## Combined Replicas
+
+A more common approach is to assign the same `name` to each Database Service
+proxying the database.
+
+Configuration for both agents in our example would be identical in this case:
+
+```yaml
+# Same config for both agents.
+db_service:
+  enabled: "yes"
+  databases:
+  - name: "postgres"
+    protocol: "postgres"
+    uri: "postgres.example.com:5432"
+```
+
+With this configuration, you will see only a single entry for the database in
+`tsh db ls`:
+
+```bash
+$ tsh db ls
+Name
+--------
+postgres
+
+$ tsh db login postgres
+$ tsh db connect postgres
+```
+
+When connecting, Teleport will randomly pick the service to connect through to
+provide some load balancing. If the selected agent is down (e.g. in case of AZ
+outage), Teleport will automatically retry to connect via other agents.
+
+<Admonition type="note" title="">
+  Teleport does not automatically reconnect if the agent goes down in the
+  middle of a database session.
```

Comment:
Does this mean that the Teleport agent doesn't automatically reconnect to the database, that the database client won't automatically reconnect to Teleport, or both?

Reply:
I meant Teleport but I agree it might be a bit confusing since client may attempt to reconnect itself, so I just removed this note.